src/data.c

```diff
@@ -254,6 +260,47 @@ const union AffineAnimCmd * const gUnknown_082FF6C0[] =
     gUnknown_082FF684,
 };
 
+static const union AnimCmd gUnknown_082FF6EC[] =
+{
+    ANIMCMD_FRAME(0, 0),
+    ANIMCMD_END,
+};
+
+static const union AnimCmd gUnknown_082FF6F4[] =
+{
+    ANIMCMD_FRAME(1, 0),
+    ANIMCMD_END,
+};
+
+static const union AnimCmd gUnknown_082FF6FC[] =
+{
+    ANIMCMD_FRAME(2, 0),
+    ANIMCMD_END,
+};
+
+static const union AnimCmd gUnknown_082FF704[] =
+{
+    ANIMCMD_FRAME(3, 0),
+    ANIMCMD_END,
+};
+
+const union AnimCmd *const gUnknown_082FF70C[] =
+{
+    gUnknown_082FF6EC,
+    gUnknown_082FF6F4,
+    gUnknown_082FF6FC,
+    gUnknown_082FF704,
+};
+
+#define SPECIES_SPRITE(species, sprite) [SPECIES_##species] = {sprite, 0x800, SPECIES_##species}
```

Comment:
instead of `0x800`, can this be `sizeof(sprite)`?

Reply:
It can't, because all gfx are decalred in graphics.s and externed as an array in graphics.h, so the compiler does not know the array size, and besides this is the size of the gfx once uncompressed.